mitodl/open-discussions
# remove problematic env variable from example

#### Pre-Flight checklist


#### What are the relevant tickets?
None

#### What's this PR do?
remove UWSGI_THREAD_COUNT from .env.example

This was causing error in initial setup:
```
web_1      | [uWSGI] getting INI configuration from uwsgi.ini
web_1      | [strict-mode] unknown config directive: thread-count
open-discussions_web_1 exited with code 1
```

#### How should this be manually tested?
Check that your local `open-discussions` instance (a) works and (b) does not have this variable set in `.env`.
